lib/tasks/geoblacklight.rake

```diff
@@ -2,7 +2,16 @@ require 'rails/generators'
 require 'generators/geoblacklight/install_generator'
 
 namespace :geoblacklight do
-  namespace :solr do
+  desc 'Run Solr and GeoBlacklight for interactive development'
+  task :server, [:rails_server_args] do |_t, args|
+    SolrWrapper.wrap(port: '8983') do |solr|
+      solr.with_collection(name: 'blacklight-core', dir: File.join(File.expand_path('../../', File.dirname(__FILE__)), 'solr', 'conf')) do
+        system "bundle exec rails s #{args[:rails_server_args]}"
+      end
+    end
+  end
+
+  namespace :index do
```

Comment:
Can we provide an alias here back to `geoblacklight:solr:seed`? That task is missing and potentially downstream clients may be using it.